<launch>
    <!-- <arg name="run_robot"   default="false"/> -->
    <arg name="run_robot"   default="true"/>

    <group if="$(arg run_robot)">
        <node name="pioneer_cross_arm" pkg="pioneer_main"     type="main_cross_arm.py"  output="screen"/>
        <!-- <node name="pioneer_cross_yolo" pkg="pioneer_yolov3"  type="yolov3_cross_arm.py"  output="screen"/> -->
        <node name="pioneer_cross_yolo" pkg="pioneer_yolov3"  type="yolov3_chip.py"     output="screen"/>
    </group>

    <!-- conventional method -->
    <!-- <node name="pioneer_cross_main" pkg="pioneer_main"    type="cross_arm_l2pc.py"    args="$(arg run_robot)" output="screen"/> -->

    <!-- deeep learning method -->
    <node name="pioneer_cross_main" pkg="pioneer_main"    type="deep_cross_arm.py"  args="$(arg run_robot)" output="screen"/>
</launch>